fs/utils.ts

```diff
@@ -5,6 +5,7 @@ import * as path from "./path/mod.ts";
  * @param src src file path
  * @param dest dest file path
  * @param sep path separator
+ * @returns boolean
```

Comment:
This line should be inferred by TypeScript. If you want to document the return value, you don’t need to provide a type here.